string got_action
string got_robot_type
string got_robot_name
string got_pose_type
string got_pose_name
float32 got_speed_scaling
float32 got_acc_scaling
float32 got_goal_tolerance